aries-site/src/examples/templates/FilterControls/FilterContext.js

```diff
@@ -0,0 +1,124 @@
+// Copyright 2021 - Hewlett Packard Enterprise Company
+
+import { createContext, useState } from 'react';
+
+const useFilter = () => {
+  // data represents the entire result set
+  const [data, setData] = useState([]);
+  // fields or columns in the data set
+  const [filterAttributes, setFilterAttributes] = useState([]);
+  // filterAttributes and associated values for which the data is to be filtered
+  const [filters, setFilters] = useState({});
+  // show/hide the layer presenting available filters
+  const [filtersLayer, setFiltersLayer] = useState(false);
+  // filteredResults are the subset of data which meets the filters criteria
+  const [filteredResults, setFilteredResults] = useState([]);
+  const [isFiltered, setIsFiltered] = useState(false);
```

Comment:
would it be safe to say if `filterResults.length` then `isFiltered` would be true? Wondering if there's opportunity to eliminate a state variable

Reply:
maybe taking back the statement actually because filtering could result in 0 matches

Reply:
You are right, there is a case where there are filters applied, but no results due to the criteria applied.